docs/UserGuide.md

```diff
@@ -156,13 +156,24 @@ Format: `delete INDEX`
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Clearing all persons or tags : `clear [t/TAG]`
+### Removing a person's field/attribute : `remove`
```

Comment:
Should we decide between field/attribute and use it consistently? Could also include the term in the glossary

Reply:
Yes we should do the glossary for UG!